huawei_solar_battery_optimization.yaml

```diff
@@ -528,7 +530,7 @@ script:
     sequence:
       - service: huawei_solar.set_maximum_feed_grid_power_percent
         data:
-          device_id: baba156a5f8b0c02d3503a0e21ce3a8e
+          device_id: "{{ states('input_text.hsbo_device_id') }}"
```

Comment:
this one should be

device_id: "{{ states('input_text.hsbo_device_id_inverter') }}"